### ISSUE TITLE
QR code downloading from 3dsthem.es not working

### ISSUE THREAD

**Issue by araeden** [ErmanSayin/Themely#76]
Everytime I try to download a theme, it fails to download (error code 0xffff). WiFi is ON and Internet is connected and working.

**Comment by araeden** [comment]
Theme exists on 3dsthem.es,and can be downloaded normally on PC.

**Comment by ErmanSayin** [comment]
what's the full error code?

**Comment by araeden** [comment]
ERROR. :/

Failed to download ZIP file

(0xffffffff)

**Comment by araeden** [comment]
Am on Luma 7.1, B9S version 1.0

**Comment by ErmanSayin** [comment]
Are you from China? Does this page show a captcha?: http://app.3dsthem.es/downloadTheme.php

**Comment by araeden** [comment]
No, I am from Pakistan. I get the error code: "!An error has occurred." No Captcha.

**Comment by ErmanSayin** [comment]
I've changed some settings on the server, please try again

**Comment by araeden** [comment]
Still not getting the Captcha, sorry.

**Comment by ErmanSayin** [comment]
Try downloading from Themely

**Comment by araeden** [comment]
Still not working in Themely 1.3.0. Tried multiple themes.

**Comment by ErmanSayin** [comment]
I can't reproduce this to fix it, sorry

**Comment by araeden** [comment]
I'll wait and see if anyone else has the same problem. it might just be something with either my o3ds specifically or my internet.

**Comment by diyasteng** [comment]
+1
n3ds B9s - Luma 8.1
updated to 1.3.1; download is stuck at 0% and is not progressing
